exonum/src/crypto.rs

```diff
@@ -143,13 +181,14 @@ pub trait CryptoHash {
     fn hash(&self) -> Hash;
 }
 
-/// Initializes the sodium library and chooses faster versions of the primitives if possible.
+/// Initializes the sodium library and automatically selects faster versions
+/// of the primitives if possible.
 ///
-/// # Panics
+/// # Panic
 ///
 /// Panics if sodium initialization is failed.
 ///
-/// # Examples
+/// # Example
```

Comment:
Plural form is used in the Rust standard library, so I prefer to follow that convention.